framework/transport/ieee1905_transport/ieee1905_transport.cpp

```diff
@@ -6,94 +6,91 @@
  * See LICENSE file for more details.
  */
 
-#include <mapf/transport/ieee1905_transport.h>
+#include <bcl/beerocks_defines.h>
+#include <mapf/common/config.h>
+
+#include "ieee1905_transport.h"
 
 #include <unistd.h>
 
-namespace mapf {
+namespace beerocks {
```

Comment:
Why? are we getting rid of the mapf namespace?

Reply:
Yup. Merging everything into BCL/BTL, so all the modules will be understand the `beerocks` namespace.